modules/oc-mirror-imageset-config-params.adoc

```diff
@@ -0,0 +1,208 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: REFERENCE
+[id="oc-mirror-imageset-config-params_{context}"]
+= Image set configuration parameters
+
+Before you can use the oc-mirror plug-in to mirror image sets, you must create an image set configuration file, which defines what images to mirror.
+
+The following table lists the available parameters for the `ImageSetConfiguration` resource.
+
+.`ImageSetConfiguration` parameters
```

Comment:
Yeah I know what you mean. Though irrespective of time, I've usually avoided doing more than 3 columns due to how tables render poorly on the portal. Not sure if this has been improved since the last time I've checked.